.github/ISSUE_TEMPLATE/bug_form.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: describe-bug
+    attributes:
+      label: Describe the bug
+      description: What behaviour was expected and what actually happened?
+      placeholder: null
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: To Reproduce
+      description: What steps would we take to reproduce the behaviour
+      placeholder: 1. Go to '...' 2. Click on '....' 3. Scroll down to '....'
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be
+        automatically formatted into code, so no need for backticks.
```

Comment:
@dianzrong how does this work? Have you tested it?

Reply:
Yes it works. You can check it out on [my repository](https://github.com/dianzrong/OpenAdapt)